polars/polars-time/src/chunkedarray/utf8/patterns.rs

```diff
@@ -4,19 +4,13 @@
 use chrono::FixedOffset;
 
 pub(super) static DATE_D_M_Y: &[&str] = &[
-    // 8-Jul-2001
-    "%v",       // 8-Jul-2001
```

Comment:
I'm suggesting to remove some of these more esoteric formats, else fail-fast becomes really complicated